static/app/views/dashboardsV2/widgetBuilder/buildSteps/sortByStep/index.tsx

```diff
@@ -44,6 +44,12 @@ export function SortByStep({
   const orderBy = queries[0].orderby;
   const maxLimit = getResultsLimit(queries.length, queries[0].aggregates.length);
 
+  const isTimeseriesChart = [
```

Comment:
```suggestion
  const isTimeSeriesChart = [
```

Reply:
So we actually use `Timeseries` in a lot of places in our code. I also want to move this into a helper method because this is copied in 3 files, but I wanted to make that change as a separate PR. I think we should just leave it as `isTimeseriesChart` for consistency right now